src/panel/category/CategoryPanel.jsx

```diff
@@ -37,28 +38,22 @@ export default class CategoryPanel extends React.Component {
   componentDidMount() {
     this.updateSearchBarContent();
     this.mapMoveHandler = listen('map_moveend', this.fetchData);
-    window.execOnMapLoaded(() => { this.fitMapAndFetch(); });
+    window.execOnMapLoaded(() => { this.fitMap(); });
   }
 
   componentDidUpdate(prevProps) {
     this.updateSearchBarContent();
-    const { bbox, poiFilters } = this.props;
 
     const panelContent = document.querySelector('.panel-content');
     if (panelContent) {
       panelContent.scrollTop = 0;
     }
 
-    // Check for a new, or changed poiFilter
-    for (const key in poiFilters) {
-      if (poiFilters[key] !== prevProps.poiFilters[key]) {
-        this.fetchData();
-        break;
-      }
-    }
-
-    if (bbox && bbox !== prevProps.bbox) {
-      window.execOnMapLoaded(() => { this.fitMapAndFetch(); });
+    if (JSON.stringify(prevProps.poiFilters) !== JSON.stringify(this.props.poiFilters)
```

Comment:
Nice trick, `_.isEqual` from lodash, (or other deep equal function), is probably more performant for bigger objects though